other/docker/slimcc/creduce.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+chibicc -c broken.c 2>&1 | grep "error"
```

Comment:
What calls this shell script? I don't see anything in this PR referencing it.

Also, why does this one use `chibicc` when everywhere else it's `slimcc`?

Reply:
Deleted.